src/audio/component.c

```diff
@@ -506,6 +516,46 @@ int comp_copy(struct comp_dev *dev)
 	return ret;
 }
 
+#ifdef SOF_PERFORMANCE_MEASUREMENTS
```

Comment:
where should this macro be defined? Should it be a Kconfig option?

Reply:
moved to Kconfig as SOF_TELEMETRY_PERFORMANCE_MEASUREMENTS